conans/cli/commands/remote.py

```diff
@@ -50,15 +50,17 @@ def output_set_user_cli(results):
 
 
 def output_remotes_json(results):
-    return json_formatter(list(results.values()))
+    return json.dumps(list(results.values()))
 
 
-@conan_subcommand(formatters={"cli": output_remote_list_cli, "json": output_remote_list_json})
+@conan_subcommand(formatters={"json": output_remote_list_json})
```

Comment:
I don't get what you have done with the formatters here. If you run "conan remote list --format json" you will get printed the regular output plus the json.

Reply:
Yes, but you get the json in stdout, the regular output is in stderr.

Reply:
So, if my bash skills are not wrong this is not working like that either:

```
> conan remote list --format json > stdout.txt 2> stderr.txt
```

```
> cat stdout.txt 
conancenter: https://center.conan.io [Verify SSL: True, Enabled: True]
[
    {
        "name": "conancenter",
        "url": "https://center.conan.io",
        "verify_ssl": true,
        "enabled": true
    }
]
```

```
> cat stderr.txt
<NOTHING>
```

Reply:
Yes, there might be some fixes to ``conan list`` command pending, adding some regular cli output (to stderr)

Reply:
I have fixed in https://github.com/conan-io/conan/pull/10232/commits/a26bc279e3b84e2b1dd353becda66efe785a96de the issues with ``conan remote`` command, that was still sending cli stuff to stdout instead of stderr.